lib/sensu-plugins-signifai/version.rb

```diff
@@ -2,7 +2,7 @@ module SensuPluginsSignifai
   module Version
     MAJOR = 1
     MINOR = 1
-    PATCH = 3
+    PATCH = 4
```

Comment:
looks like you missed reverting the version but I can get this for you post acceptance prior to release.

Reply:
```suggestion
    PATCH = 3
```